index.js

```diff
@@ -1,9 +1,16 @@
 (function() {
-
   var RATE_LIMIT = 25;
 
+  /* Finding browser event names for hidden window event */
+  var hidden, visibilityChange;
+  ['',"moz","ms","webkit"].some(function(prefix) {
```

Comment:
tested this in FF, Chrome and IE11, and all of them work without prefix, so let's remove this and just use the proper events and state names in the code